modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCachePartitionExchangeManager.java

```diff
@@ -476,6 +489,19 @@ else if (m instanceof GridDhtPartitionDemandLegacyMessage) {
                 });
             }
         }
+
+        MetricRegistry mreg = cctx.kernalContext().metric().registry(PME_METRICS);
+
+        mreg.register(CURRENT_PME_DURATION,
```

Comment:
Let's rename to `PME_DURATION`

Reply:
@nizhikov {{PME_DURATION}} is already exist. It's the histogram.